docs/docs/styled-components.md

```diff
@@ -112,7 +112,29 @@ export default () => (
 )
 ```
 
-### Enabling user stylesheets with a stable class name
+## Creating Global Styles
+
+Styled-components are primarily used for a single CSS class that is isolated from other components. In some cases, you want to override global styling — for example, the default margins of your `body` element. Styled-components has your back. You can use the `createGlobalStyle` to accomplish this. We advise using `createGlobalStyle` in Layout components, which are shared over multiple pages rather than using it on a single page.
```

Comment:
```suggestion
Styled-components are primarily used for a single CSS class that is isolated from other components. In some cases, you want to override global styling — for example, the default margins of your `body` element. Styled-components has your back. You can use the `createGlobalStyle` API to accomplish this. It's advised to use `createGlobalStyle` in [Layout components](/docs/layout-components/), which are shared over multiple pages rather than on a single page.
```
Three things here: 
- adding the word "API" since it seemed like it was missing (removing "the" would work too)
- following the [Gatsby Style Guide](https://www.gatsbyjs.org/contributing/gatsby-style-guide/#use-you-as-the-pronoun) to focus on the reader
- adding a link to a doc on Layout components so people can learn more about them!